docs/_documentations/overview.md

```diff
@@ -1,29 +1,83 @@
+
+### How does Codewind help me develop containerized applications?
+Codewind improves your inner loop experience, enabling you to create a microservice quickly, rapidly iterate on changes, and make improvements to performance thanks to the following features:
+
+1. **Access to preconfigured containerized project templates** Codewind provides preconfigured, Docker-containerized [project templates](mdt-vsc-workingwithtemplates.html) covering several languages, including [Node.js](https://nodejs.dev/), [Java](https://www.java.com/), [Python](https://www.python.org/) and [Swift](https://swift.org/), and several frameworks, such as [Express](https://expressjs.com/), [Spring Boot](https://spring.io/projects/spring-boot), and [Open Liberty](https://openliberty.io/). [Quickly create and deploy microservices](https://www.youtube.com/watch?v=zKMggp10gq4&t=12s) in languages and frameworks that you're familiar with. Modify the preconfigured projects to develop your customised microservice.
+
+2. **Ability to add your own standardised project templates** Create your own [project templates](mdt-vsc-workingwithtemplates.html) and your development teams can access them within their IDE. 
+
+3. **Ability to add your own applications** Access the seamless container development and deployment experience on your IDE that Codewind offers for your current projects.
+
+4. **Auto-build-and-run on code changes** As you develop, Codewind automatically pushes your code changes to your container as efficiently as possible. 
+
+5. **Ability to toggle between auto-build and manual build settings** Have control over when you want to implement Codewind's auto-build feature.
+
+6. **Project details displayed in a user-friendly panel on your IDE** Use Codewind's Project Overview interface to instantly view the endpoint, container ID, location on disk, build and run status, and ports.
+
+7. **Access application, build and container logs and run commands from within the container without leaving your IDE** Codewind integrates with your IDE's terminal window and PROBLEMS and OUTPUT panels for a smooth test and debug experience. Additionally, you can open a shell session *inside* your container with a simple click.
+
+8. **Support for your IDE's debug features**<sup>*</sup> Full compatibility with the debug functionality of your IDE.
+
+9. **Visualize your container application's footprint such as CPU and memory usage** <sup>*</sup> Codewind's Application Metrics Dashboard exposes your microservice's core metrics, including HTTP(S) request and throughput information, as well as CPU usage, memory footprint, and heap.
+
+10. **Integrated load-testing** <sup>*</sup> Configure and run load tests against your microservices using [Codewind's performance dashboard](https://www.youtube.com/watch?v=nfJt3f5TUvc). Use the graphs to compare your repeated load test results and iterate on performance improvements. 
+
+<sup>*</sup> where supported.
```

Comment:
This file is the only one with superscripts. Could we remove them and instead mention "where supported" as part of the titles or descriptions?